.github/workflows/ios-ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: select Xcode version
-        run: sudo xcode-select -switch /Applications/Xcode_11.7.app
+        run: sudo xcode-select -switch /Applications/Xcode_12.4.app
```

Comment:
Should we change this to 12.5 now?

Reply:
Good question! I will need to check as the reason for this was to align with the CI of our Backpack App and also the Skyscanner App which does not yet support Xcode 12.5 so making sure conditions are similar

Reply:
We are now using xcode 12.5 everywhere 👍